hebiros_advanced_examples/src/omnibase/Readme.md

```diff
@@ -0,0 +1,83 @@
+# Overview
+
+These files are used to interface with the omnibase robot base. This documentation provides instruction to run each of the nodes, as well as information on the ROS messages used by each file.
+
+**Important: to run any of these commands, you will need to run `source devel/setup.sh` from your catkin workspace first**
+
+
+# Initial Setup
+
+In a new directory (or your existing catkin workspace), run the following:
+
+```
+mkdir src
+cd src
+git clone https://github.com/HebiRobotics/HEBI-ROS.git
+git clone https://github.com/HebiRobotics/hebi_cpp_api_ros
+cd HEBI-ROS
+git checkout develop
+cd ..
+catkin_init_workspace
+cd ..
+```
+
+To build the catkin workspace, run the following commands:
+```
+catkin_make clean
+catkin_make
+source devel/setup.bash
+```
+
+# Running the code
+
+## Omnibase nodes:
+
+To run the omnibase odometry node using the keyboard teleop code, use this command:
+```
+roslaunch hebiros_advanced_examples example_omnibase_odometry.launch 
+```
+
+To run the omnibase odometry node using the Mobile I/O app, use this command:
+```
+roslaunch hebiros_advanced_examples omnibase_navigation_mobile_IO.launch 
+```
+
+To run the omnibase free_roam node, use this command:
+```
+roslaunch hebiros_advanced_examples example_omnibase_free_roam.launch
+```
+
+## Mobile I/O app interface:
+
+To run just the Mobile I/O app interface, run the following command:
+```
+python mobile_io_controller.py
+```
+*Note: In order to run this command, there must be an existing rosmaster node running on the system*
+
+
+# Rostopic details for each file:
+
+## omnibase_free_roam
+
+The free_roam node subscribes to the `/cmd_vel` rostopic; the node converts the velocity messages from the topic to movement commands for the omnibase robot.
+
+
+## omnibase_odometry
+
+The odometry node subscribes to the `/joint_state` rostopic; on this topic, the node recieves position data for each of the omnibase actuators.
```

Comment:
I might clarify "receives position commands" instead of "data", as data could be command or feedback.

Also, i before e except after c (and a million other special cases).  But it is "receives".